proof/refine/ARM_HYP/ArchInvsDefs_H.thy

```diff
@@ -63,33 +65,35 @@ definition tcb_vcpu_refs' :: "machine_word option \<Rightarrow> (obj_ref \<times
   "tcb_vcpu_refs' t \<equiv> set_option t \<times> {TCBHypRef}"
 
 definition tcb_hyp_refs' :: "arch_tcb \<Rightarrow> (obj_ref \<times> reftype) set" where
-  "tcb_hyp_refs' t \<equiv> tcb_vcpu_refs' (AARCH64_H.atcbVCPUPtr t)"
+  "tcb_hyp_refs' t \<equiv> tcb_vcpu_refs' (ARM_HYP_H.atcbVCPUPtr t)"
 
 definition vcpu_tcb_refs' :: "obj_ref option \<Rightarrow> (obj_ref \<times> reftype) set" where
   "vcpu_tcb_refs' t \<equiv> set_option t \<times> {HypTCBRef}"
 
 definition refs_of_a' :: "arch_kernel_object \<Rightarrow> (obj_ref \<times> reftype) set" where
   "refs_of_a' x \<equiv> case x of
-     AARCH64_H.KOASIDPool asidpool \<Rightarrow> {}
-   | AARCH64_H.KOPTE pte \<Rightarrow> {}
-   | AARCH64_H.KOVCPU vcpu \<Rightarrow> vcpu_tcb_refs' (AARCH64_H.vcpuTCBPtr vcpu)"
+     ARM_HYP_H.KOASIDPool asidpool \<Rightarrow> {}
+   | ARM_HYP_H.KOPDE pte \<Rightarrow> {}
```

Comment:
Should be `ARM_HYP_H.KOPDE pde`

Reply:
Good idea with `_`, we should use that when we don't care what the value is.